README.md

```diff
@@ -2,42 +2,339 @@
+
+```julia
+
+# Initialize phases on the particles
+function init_phases!(phase_ratios, xci, radius)
+    ni      = size(phase_ratios.center)
+    origin  = 0.5, 0.5
+    
+    @parallel_indices (i, j) function init_phases!(phases, xc, yc, o_x, o_y)
+        x, y = xc[i], yc[j]
+        if ((x-o_x)^2 + (y-o_y)^2) > radius^2
+            JustRelax.@cell phases[1, i, j] = 1.0
+            JustRelax.@cell phases[2, i, j] = 0.0
+        
+        else
+            JustRelax.@cell phases[1, i, j] = 0.0
+            JustRelax.@cell phases[2, i, j] = 1.0
+        end
+        return nothing
+    end
+
+    @parallel (JustRelax.@idx ni) init_phases!(phase_ratios.center, xci..., origin...)
+end
+```
+Initialisation of the physical domain and the grid. As `JustRelax.jl` is utilising [ImplicitGlobalGrid.jl](https://github.com/omlins/ImplicitGlobalGrid.jl), the grid can be `MPIAWARE` through setting the grid steps in x- and y- direction to ` di = @. li / (nx_g(),ny_g())`. This makes it a global grid and the grid steps are automatically distributed over the MPI processes. 
+
+```julia
+
+# MAIN SCRIPT --------------------------------------------------------------------
+function main(igg; nx=64, ny=64, figdir="model_figs")
+
+    # Physical domain ------------------------------------
+    ly       = 1e0          # domain length in y
+    lx       = ly           # domain length in x
+    ni       = nx, ny       # number of cells
+    li       = lx, ly       # domain length in x- and y-
+    di       = @. li / ni   # grid step in x- and -y
+    origin   = 0.0, 0.0     # origin coordinates
+    xci, xvi = lazy_grid(di, li, ni; origin=origin) # nodes at the center and vertices of the cells
+    dt       = Inf 
+```
+Initialisation of the Rheology utilising [GeoParams.jl](https://github.com/JuliaGeodynamics/GeoParams.jl). The Rheology can be taylored to the specific problem with different creep laws and material parameters (see [GeoParams.jl](https://github.com/JuliaGeodynamics/GeoParams.jl)) or the miniapps in the [convection folder](miniapps/convection).
+
+```julia
+    # Physical properties using GeoParams ----------------
+    τ_y     = 1.6           # yield stress. If do_DP=true, τ_y stand for the cohesion: c*cos(ϕ)
+    ϕ       = 30            # friction angle
+    C       = τ_y           # Cohesion
+    η0      = 1.0           # viscosity
+    G0      = 1.0           # elastic shear modulus
+    Gi      = G0/(6.0-4.0)  # elastic shear modulus perturbation
+    εbg     = 1.0           # background strain-rate
+    η_reg   = 8e-3          # regularisation "viscosity"
+    dt      = η0/G0/4.0     # assumes Maxwell time of 4
+    el_bg   = ConstantElasticity(; G=G0, Kb=4)
+    el_inc  = ConstantElasticity(; G=Gi, Kb=4)
+    visc    = LinearViscous(; η=η0) 
+    pl      = DruckerPrager_regularised(;  # non-regularized plasticity
```

Comment:
Cite one of Thibault's plascitiy papers (e.g. https://agupubs.onlinelibrary.wiley.com/doi/pdf/10.1029/2022GC010675)

Reply:
see L:175

Reply:
Can you share the permalink of that line? I don't see anything